### PR TITLE
add basic travis config

### PR DESCRIPTION
**Pull request by iiwo**
- fix incompatible migration
- add apprisal config
- add basic travis config
- update gitignore & readme

travis passing on fork build: https://travis-ci.org/iiwo/fast-versioning/builds/560437492

### CODE REVIEW

File: db/migrate/20190514134359_add_unique_index_to_version_id_name.rb [hunk 1]
```diff
@@ -1,7 +1,5 @@
 class AddUniqueIndexToVersionIdName < ActiveRecord::Migration[5.0]
-  disable_ddl_transaction!
-
   def change
-    add_index :fast_versioning_fast_versions, %i[version_id name], unique: true, algorithm: :concurrently
+    add_index :fast_versioning_fast_versions, %i[version_id name], unique: true
```
Comment (jakedouglas) [inline db/migrate/20190514134359_add_unique_index_to_version_id_name.rb:3]: :+1: Ran into that locally.

File: Appraisals [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  gem 'paper_trail', '~> 4.0'
+end
+
+appraise 'ar-5.0_pt-5.0' do
+  gem 'activerecord', '~> 5.0.0'
+  gem 'paper_trail', '~> 5.0'
+end
+
+appraise 'ar-5.1_pt-4.0' do
+  gem 'activerecord', '~> 5.0.0'
```
Comment (jakedouglas) [inline Appraisals]: Should this be `~> 5.1.0`?

File: Appraisals [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  gem 'paper_trail', '~> 4.0'
+end
+
+appraise 'ar-5.1_pt-5.0' do
+  gem 'activerecord', '~> 5.0.0'
```
Comment (jakedouglas) [inline Appraisals]: Likewise?

File: Appraisals (hunk 2 above)
Comment (iiwo) [reply Appraisals]: oh 🤦‍♂, good catch, thanks!

**Review by jakedouglas — APPROVED** [review]
I haven't used `appraisal` before but this all makes sense to me.

**Pull request merged**